packages/preview/simple-bupt-report/0.1.1/template/main.typ

```diff
@@ -0,0 +1,54 @@
+#import "@preview/simple-bupt-report:0.1.1": experiment-report
+
+#show: doc => experiment-report(
+  title: "《信号处理实验》实验报告",
+  semester: "2024-2025学年第二学期",
+  class: "2023211113",
+  name: "张三",
+  student-id: "2021123456",
+  date: "2024年4月14日",
+  doc
+)
+
+= 实验一 频谱泄露实验
+
+== 实验目的
+
+（1）掌握MATLAB的基本使用方法；
+
+（2）其他实验目的；
+
+（3）xxx。
+
+== 实验原理
+=== 原理一
+写一些使用到的原理，简略得当，过长将扣分。
+插入公式，建议大家使用MathType，或者采用WPS自带的公式编辑器，如图1所示。
+=== 原理二
+
+
+== 实验步骤
+=== 码上使用过程
+写明码上问答过程及使用情况。
+===	Matlab实验过程
+如@bupt-logo ，写明实验步骤，简明扼要。 
+
+#figure(
+  image("../assets/bupt-logo.jpg",width: 30%),
```

Comment:
This image should be moved in the `template` subdirectory if you want to load it from this file.